CONTRIBUTING.md

```diff
@@ -37,3 +37,10 @@ Attempt to make your test cases produce clearly-differentiable results regarding
 If there are additional interesting engine variations you are testing, for example older versions or engines not in the list, feel free to add more columns. If you can't test certain engines, leave a question mark in that cell, and others on the thread can help fill them in.
 
 For examples of this kind of investigation in action, see [whatwg/html issue #775](https://github.com/whatwg/html/issues/775#issuecomment-190796607) or [whatwg/html issue #1087](https://github.com/whatwg/html/issues/1087#issue-150128324).
+
+## Principles
```

Comment:
Seems like a reasonable place to also mention https://whatwg.org/working-mode and maybe https://whatwg.org/faq#adding-new-features.